src/main/java/uk/gov/hmcts/reform/sscs/domain/wrapper/Rate.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.sscs.domain.wrapper;
+
+public enum Rate {
```

Comment:
Think this can be removed

Reply:
Same here, used by OnlineHearing data object